docs/database/elasticsearch.md

```diff
@@ -57,7 +57,9 @@ Tracing instrumentations that do so, MUST also set `http.request.method_original
 
 **[3]:** For network calls, URL usually has `scheme://host[:port][path][?query][#fragment]` format, where the fragment is not transmitted over HTTP, but if it is known, it SHOULD be included nevertheless.
 `url.full` MUST NOT contain credentials passed via URL in form of `https://username:password@www.example.com/`. In such case username and password SHOULD be redacted and attribute's value SHOULD be `https://REDACTED:REDACTED@www.example.com/`.
-`url.full` SHOULD capture the absolute URL when it is available (or can be reconstructed). Sensitive content provided in `url.full` SHOULD be scrubbed when instrumentations can identify it.
+`url.full` SHOULD capture the absolute URL when it is available (or can be reconstructed).
+Query string values SHOULD be redacted by default and replaced by the value `REDACTED`, e.g. `https://www.example.com/path?abc=REDACTED&xyz=REDACTED` (the query string keys SHOULD be preserved).
+Instrumentation MAY provide a configuration option to capture the full query string without any redaction.
```

Comment:
@AlexanderWert @trisch-me do you know if sensitive data can be present in elasticsearch query string values?

Reply:
it depends on data, for example if email address is saved in ES, one could also query user by an email address. or any other PII data saved in DB